kornia/contrib/models/sam/architecture/tiny_vit.py

```diff
@@ -0,0 +1,471 @@
+# https://github.com/microsoft/Cream/blob/8dc38822b99fff8c262c585a32a4f09ac504d693/TinyViT/models/tiny_vit.py
+# https://github.com/ChaoningZhang/MobileSAM/blob/01ea8d0f5590082f0c1ceb0a3e2272593f20154b/mobile_sam/modeling/tiny_vit_sam.py
+# NOTE: make this available as an image classifier?
```

Comment:
makes sense to me :)

Reply:
If we make this available as an image classifier also, any idea where I should put `tiny_vit.py`? I'm thinking under kornia/contrib/models

Reply:
yeah kornia.contrib.models